ivy/container/linear_algebra.py

```diff
@@ -1263,16 +1312,58 @@ def slogdet(
         to_apply: bool = True,
         prune_unapplied: bool = False,
         map_sequences: bool = False,
-        *,
-        out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.slogdet. This method simply wraps
+        the function, and so the docstring for ivy.slogdet also applies to this method
+        with minimal changes.
+
+        Parameters
+        ----------
+        self
+            input container having shape (..., M, M) and whose innermost two dimensions
+            form square matrices. Should have a floating-point data type.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+
+        Returns
+        -------
+        ret
+            This function returns container containing NamedTuples.
+            Each NamedTuple of output will have -
+                sign:
+                An array of a number representing the sign of the determinant of each
+                square.
+
+                logabsdet:
+                An array of the natural log of the absolute value of the determinant of
+                each square.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([[1.0, 2.0], \
```

Comment:
could you please add a `b` element here as well? 😃

Reply:
@ash-ishh this comment was marked as resolved even though it was not addressed 😅